cli/setup.py

```diff
@@ -19,7 +19,8 @@
     include_package_data=True,
     install_requires=[
         'click',
-        'pyyaml'
+        'pyyaml',
+        'ruamel.yaml<0.15'
```

Comment:
do we need to strictly use a version < 0.15? Also, I was under the impression that ruamel replaces pyyaml, why do we need to leave the two?